doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -564,6 +564,70 @@ The fully expanded example above (without environment variables) looks like this
 
 Note: you must have "Add Dataset" permission in the given collection to invoke this endpoint.
 
+List Featured Collections for a Dataverse Collection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Lists the aliases of the featured collections of a given Dataverse collection identified by ``id``:
```

Comment:
Can you add that the output is a JSON array of the alias strings?

Reply:
does this look ok? https://github.com/IQSS/dataverse/pull/10459/commits/98cd85cbb6fbf03f3c7217a755b8a27dbb6328b2